docs/victory-axis/docs.md

```diff
@@ -22,6 +22,13 @@ The `tickCount` prop specifies approximately how many ticks should be drawn on t
 
 *default:* `tickCount={5}`
 
+### fixLabelOverlap
+
+When the `fixLabelOverlap` boolean prop is set to `true` axis tick labels and their associated ticks
```

Comment:
start of first sentence could be simplified to `When true, axis tick labels and their associated ticks...`.